internal/neofs/neofs.go

```diff
@@ -606,3 +492,66 @@ func (x *NeoFS) SystemDNS(ctx context.Context) (string, error) {
 
 	return domain, nil
 }
+
+// AuthmateNeoFS is a mediator which implements authmate.NeoFS through NeoFS.
```

Comment:
```suggestion
// AuthmateNeoFS is a mediator which implements authmate.NeoFS through pool.Pool.
```
since `NeoFS` appears only in the implementation and does not stick out (`NewAuthmateNeoFS` accepts `pool.Pool`).